src/build-config/webpack.js

```diff
@@ -1,4 +1,4 @@
-exports.registerSvgWebpackLoader = (config) =>
+export const registerSvgWebpackLoader = (config) =>
```

Comment:
We should write it as TS now, so we don't need any new infrastructure.
Type-wise that might get hard, but using simplified types here is fine.